relay/ethToElrond/constants.go

```diff
@@ -3,6 +3,22 @@ package ethToElrond
 const (
 	// GetPending is the step definition for the pending transactions check
 	GetPending = "get pending transactions"
+
 	// ProposeTransfer is the step definition for the propose transfer operation
 	ProposeTransfer = "propose transfer"
+
+	// WaitForSignaturesForProposeTransfer is the step definition for the signature gathering process for propose transfer
+	WaitForSignaturesForProposeTransfer = "wait for signatures for transfer propose"
```

Comment:
I think we should use Present Continuous Tense for these steps. "waiting signatures for transfer propose", "executing transfer" and so on